app/core/Multichain/test/utils.test.ts

```diff
@@ -87,6 +91,22 @@ const mockBTCAccount: InternalAccount = {
   type: BtcAccountType.P2wpkh,
 };
 
+const mockSolAccount: InternalAccount = {
+  address: SOL_ADDRESSES,
```

Comment:
Nit: Should this be singular instead? 😅 (we would need to change the declaration too)
Nit: Also, other mock addresses are prefixed by `MOCK_`, we could update this to align everything.

```suggestion
  address: SOL_ADDRESS,
```

Reply:
good point.